### DISCUSSION TITLE
graphql/codegen/plugins/c-sharp/c-sharp-operations

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#9047] [Docs Discussions]
# graphql/codegen/plugins/c-sharp/c-sharp-operations

GraphQL Code Generator Documentation

https://the-guild.dev/graphql/codegen/plugins/c-sharp/c-sharp-operations

**Comment by mariusmuntean** [comment]
We're using the c-sharp plugin (which doesn't seem to have a documentation page) in our test project. It helps us test our multiple GraphQL APIs.
Recently we found out that it doesn't support GraphQL union types and neither does c-sharp-operations. Both plugins have been updated half a year ago.
Would it be possible to add support for union types?

**Reply by dimaMachina** [reply]
PR welcome
